addons/bestja_detailed_reports/views/project.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="bestja_project_form">
+            <field name="model">bestja.project</field>
+            <field name="inherit_id" ref="bestja_requests.bestja_project_form"/>
+            <field name="arch" type="xml">
+                <field name="request_template" position="after">
```

Comment:
Again. Can not reference elements from modules that are independent of your module and may not be present.  You should use `<page id="page_reports">` instead to just append this at the bottom of the page (tab).